sdk/core/azure-core/src/main/java/com/azure/core/util/polling/Poller.java

```diff
@@ -122,11 +129,13 @@
      *     operation, it should be handled by the client library and return a valid {@link PollResponse}. However if
      *     the poll operation returns {@link Mono#error(Throwable)}, the {@link Poller} will disregard it and continue
      *     to poll.
+     * @param fetchResultOperation the operation to be called to fetch final result after polling has been completed.
      * @throws IllegalArgumentException if {@code pollInterval} is less than or equal to zero and if
      *     {@code pollInterval} or {@code pollOperation} are {@code null}
      */
-    public Poller(Duration pollInterval, Function<PollResponse<T>, Mono<PollResponse<T>>> pollOperation) {
-        this(pollInterval, pollOperation, null, null);
+    public Poller(Duration pollInterval, Function<PollResponse<T>, Mono<PollResponse<T>>> pollOperation,
```

Comment:
Is it required to supply a final fetch operation? I can see the use case in create certificate but we don't want to fetch the blob when it's complete copying.